app/src/main/kotlin/com/hippo/ehviewer/ui/screen/GalleryDetailContent.kt

```diff
@@ -774,15 +777,14 @@ private fun List<GalleryTagGroup>.getArtistTag(): String? {
 
 context(Context)
 @Composable
-private fun GalleryDetail?.collectPreviewItems(prefetchDistance: Int) = rememberInVM(this) {
+private fun GalleryDetail?.collectPreviewItems() = rememberInVM(this) {
     val detail = this@collectPreviewItems ?: return@rememberInVM emptyFlow()
     val pageSize = detail.previewList.size
     val pages = detail.pages
     val previewPagesMap = detail.previewList.associateBy { it.position } as MutableMap
     Pager(
         PagingConfig(
             pageSize = pageSize,
-            prefetchDistance = prefetchDistance,
```

Comment:
`pageSize` is up to 400, I don't think we should use that as `prefetchDistance`.

Reply:
The point is, the user may not move to the next page if pageSize is 400. It's not expensive but just unnecessary.